.github/workflows/build-test-package.yml

```diff
@@ -135,7 +135,7 @@ jobs:
     strategy:
       max-parallel: 2
       matrix:
-        python-version: [36, 37, 38, 39]
+        python-version: [37, 38, 39, 310]
```

Comment:
310 might need to be quoted, see https://github.com/InsightSoftwareConsortium/napari-itk-io/pull/5#discussion_r834312034
That setup might be different, so that might not apply here.

Reply:
Good thought, I believe the setup is different here though. We pass the `python-version` string to our build script so that it gets the correct version, i.e. as "cp310"; see [https://github.com/InsightSoftwareConsortium/ITKSplitComponents/blob/master/.github/workflows/build-test-package.yml#L162](https://github.com/InsightSoftwareConsortium/ITKSplitComponents/blob/master/.github/workflows/build-test-package.yml#L162). By contrast the napari repo is using the python version directly to inform a Github Actions job, see [https://github.com/InsightSoftwareConsortium/napari-itk-io/blob/main/.github/workflows/test_and_deploy.yml#L31](https://github.com/InsightSoftwareConsortium/napari-itk-io/blob/main/.github/workflows/test_and_deploy.yml#L31).

The fact the the pipeline passes as-is is also useful evidence in favor of the status quo. 😄